### ISSUE TITLE
Document link for Create Views, Triggers and any possible extn to create ER Diagram 

### ISSUE THREAD

**Issue by IoTFier** [sqlitebrowser/sqlitebrowser#1861]
1> Is there a document link that explains how to create views and triggers?
2> Is it possible to create ER diagram using the tables created in sqlitebrowser that will also show PK, FK, Indexes etc ? If not, any extension or other alternative to do so?
3> How do I change the schema name from main to my schema name? Is there a way to provide more than one schema names to choose from in the drop down when we create a new table?

Thanks

**Comment by scottfurry** [comment]
1) I'll leave to someone else to answer
2) ER diagrams are just cool... My head just jumped to some kind of graphviz or DOT language(graphics library not MS) to show entity relationships. Must ponder a little more. I would vote for this and maybe should be a separate feature request.
3) [This](https://www.sqlite.org/lang_attach.html) leads me to think schema names are not changeable within SQLite (governing library). From the page:

> The schema-names 'main' and 'temp' refer to the main database and the database used for temporary tables. The main and temp databases cannot be attached or detached.

Basically, SQLite is not a full-fledged RDBMS system like PostgreSQL, MySQL, et al. It doesn't necessarily have schemes. When a scheme name is called for(i.e. from SQL standard), IIRC and am reading this right, SQLite will use database file name. But I could be wrong on this point.

**Comment by mgrojo** [comment]
1. Already answered by @scottfurry about views. About triggers: currently you cannot create or edit triggers from the GUI. Use the Execute SQL tab: https://sqlite.org/lang_createtrigger.html
2. Not currently implemented. See the feature request:  #913
3. Schemas in SQLite are related to ATTACH DATABASE: https://www.sqlite.org/lang_attach.html You can attach databases from the GUI in DB4S. Once a DB is attached, you can create tables in that schema or move tables from the main schema to the attached schema (press the Advanced button in the Create/Edit Table dialog).

**Comment by mgrojo** [comment]
@IoTFier I hope we have helped you. I'll close this now, but you can still comment on it if you'd like.
